modules/rpc/handler.go

```diff
@@ -26,6 +26,7 @@ import (
 	"go.uber.org/fx"
 	"go.uber.org/fx/internal/fxcontext"
 	"go.uber.org/fx/service"
+	"go.uber.org/fx/uauth"
```

Comment:
Why not just `auth` here? Might make sense to keep the branding at the top-level, e.g. FX, but then have all the parts *inside* FX be named in a boring and straightforward way.